build/PackageDiffIgnore.xml

```diff
@@ -1874,6 +1874,16 @@
 		</Fields>
 
 		<Properties>
+			<!-- BEGIN templated-parent rework -->
+
+			<!-- DependencyObject (codegen for all descendants) -->
+			<Member fullName=".*DependencyObject .*::TemplatedParent" reason="Not in UWP" isRegex="true" />
```

Comment:
We may consider at least keeping this property for backcompat. Setter should probably be no-op.